email.go

```diff
@@ -138,9 +139,10 @@ const (
 // NewMSG creates a new email. It uses UTF-8 by default. All charsets: http://webcheatsheet.com/HTML/character_sets_list.php
 func NewMSG() *Email {
 	email := &Email{
-		headers:  make(textproto.MIMEHeader),
-		Charset:  "UTF-8",
-		Encoding: EncodingQuotedPrintable,
+		headers:                   make(textproto.MIMEHeader),
+		Charset:                   "UTF-8",
+		Encoding:                  EncodingQuotedPrintable,
+		DuplicateRecipientAllowed: false,
```

Comment:
Not need to add this line here because it's the default value

Reply:
Sure, removed it.